index.js

```diff
@@ -1207,6 +1207,10 @@ ArrayTome.prototype.shift = function () {
 		diff(this, 'shift');
 	}
 
+	if (ObjectTome.isObjectTome(out)) {
```

Comment:
`UndefinedTome` breaks hard when re-conjured without a parent so we need to do it for specific cases. Before we were doing `valueOf()` on returned values so I think a `untome` would actually make sense.